### DISCUSSION TITLE
r167.2 release download blocked by Bitdefender Security Antivirus

### DISCUSSION THREAD

**Discussion by dpluigi** [Purfview/whisper-standalone-win#160] [Q&A] [answered]
The latest Whisper-standalone-win release download was blocked by Bitdefender Security Antivirus   Its "online threat prevention" detected the webpage as infected.  Has this been noted and was it the trigger?

I use Subtitle Editor is Whisper-Standalone included with it useable outside of SE and/or the same version as from Purfiew?

I want to work on audio file excerpts from a movie I am making the original language (Czech) subtitles rather than from with SE.

Thanks for your  help and input

**Comment by Purfview** [comment, marked as answer]
> The latest Whisper-standalone-win release download was blocked by Bitdefender Security Antivirus Its "online threat prevention" detected the webpage as infected. Has this been noted and was it the trigger?

Dunno, ask Bitdefender's support.
Maybe you need to update it's definition files as it's not flagging it on Virustotal.

Here is related thread -> [Standalone versions analyzed with virustotal ](https://github.com/Purfview/whisper-standalone-win/discussions/91)

> I use Subtitle Editor is Whisper-Standalone included with it useable outside of SE and/or the same version as from Purfiew?

SE v4.0.3 downloads `r167.2`, which is the current version at the moment.
